### PR TITLE
Docs for syn port scanner

### PR DESCRIPTION
**Pull request by d78ui98**
documentation for syn portscan

## Verification

List the steps needed to make sure this thing works

- [x] Start `msfconsole`
- [x] `use auxiliary/scanner/portscan/syn`
- [x] `info -d`
- [x] **Verify** the docs load and look correct

### CODE REVIEW

**Comment by h00die** [comment]
I cleaned up the title and submission info. i'll tackle this one in a few hours.

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+## Description
+  
+This module will attempt to initiate a three-way handshake with every
```
Comment (wvu-r7) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: Every...?

Comment (h00die) [reply documentation/modules/auxiliary/scanner/portscan/syn.md]: Syn scanner three way handshake?!

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Vulnerable Application
+
+  Any reachable TCP endpoint is a potential target.
+
+## Options
+
+  **PORTS**
+  
+  This is the list of ports to test for TCP Scan on each host.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: This is the list of TCP ports to test on each host.

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  Formats like  `1-3`, `1,2,3`, `1,2-3`, etc. are all supported.Default
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: supported. Default (add a space)

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  options is to scan `1-10000` ports.
+
+  **TIMEOUT**
+  
+   Maximum time (seconds) to wait for a response. The default value is 500.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: `500`

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+ 
+  **ConnectTimeout**
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: this doesn't exist in syn scanner.

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  connection. Default value if 10.
+ 
+  **VERBOSE**
+  
+  Gives detailed message about the scan of all the ports. It also shows the
+  ports that were closed.
+
+## Verification Steps
+
+  1. Do: `use auxiliary/scanner/portscan/tcp`
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: syn?

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+  2. Do: `set RHOSTS [IP]`
+  3. Do: `set RPORT [PORTS]`
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: PORTS

File: documentation/modules/auxiliary/scanner/portscan/syn.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  4. Do: `run`
+  5. If any of the TCP ports were open they will be discovered, status will be printed indicated as such.
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/portscan/syn.md]: indicating

**Review by h00die — CHANGES_REQUESTED** [review]

**Comment by d78ui98** [comment]
I have made the requested changes. Thanks @h00die  @wvu-r7

**Comment by h00die** [comment]
# Release Notes
Docs for syn port scanner.

**Comment by h00die** [comment]
@d78ui98 Thank you for the submission.  I had one minor correction https://github.com/rapid7/metasploit-framework/commit/ddfa1d722bb98f21992b6a151e856e24ce44fb06 but didn't want to push this back any further.

**Pull request merged**